spec/lib/efile/nj/nj_retirement_income_helper_spec.rb

```diff
@@ -0,0 +1,132 @@
+require 'rails_helper'
+
+RSpec.describe Efile::Nj::NjRetirementIncomeHelper do
+  describe ".calculate_maximum_exclusion" do
+    [
+      { total_income: 99_000, expected: 75_000 },
```

Comment:
[pebble] for all these cases in this whole test file, could we test edge cases on both sides of each boundary? It'll be a lot more test cases to write but worth it rather than just having example cases from the middle of the range. Right now, if we mistakenly changed the magic numbers in our code, some of this could still pass.

I'm thinking:
- 100,000 (first column)
- 100,001 (second column edge 1)
- 125,000 (second column edge 2)
- 125,001 (third column edge 1)
- 150,000 (third column edge 2)
- 150,001 (beyond case)